apparmor.d/groups/virt/k3s

```diff
@@ -0,0 +1,176 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2022 Jeroen Rijken
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}{local/,}bin/k3s
+profile k3s @{exec_path} flags=(complain) {
+  include <abstractions/base>
+  include <abstractions/ssl_certs>
+  include <abstractions/nameservice-strict>
+  include <abstractions/disks-read>
+
+  capability chown,
+  capability dac_override,
+  capability dac_read_search,
+  capability net_admin,
+  capability syslog,
+  capability sys_admin,
+  capability sys_resource,
+
+  ptrace peer=@{profile_name},
+  ptrace (read) peer=unconfined,
+
+  network inet    dgram,
+  network inet6   dgram,
+  network inet    stream,
+  network inet6   stream,
```

Comment:
```suggestion
  network inet dgram,
  network inet6 dgram,
  network inet stream,
  network inet6 stream,
```

Only the is executable rules (rPx, rix...) are (sometime) aligned to the right. In case of doubt, just do not do it.